integration-tests/integration/dpsuser.spec.js

```diff
@@ -1,3 +1,5 @@
+const path = require('path')
+const parse = require('csv-parse')
```

Comment:
There don't seem to be many dependencies in this project so it felt kinda wrong adding some...

Reply:
sound fine to me